core/src/test/scala/unit/kafka/raft/RaftManagerTest.scala

```diff
@@ -306,6 +309,113 @@ class RaftManagerTest {
 
   }
 
+  class ReconfigurationTestContext(
```

Comment:
If you agree with my comments in `RaftManager`, I am okay removing these tests. I think when we have the real implementation will want to add a `KafkaRaftClientReconfigTest`, similar to `KafkaRaftClientTest` and `KafkaRaftClientSnapshotTest` that uses `RaftClientTestContext` to test `KafkaRaftClient`.

Reply:
ok